doc/README.md

```diff
@@ -1,11 +1,17 @@
 # ROS2 Navigation System Documentation
 This is where the ROS2 Navigation System documentation is being collected and vetted.
 
+# Use Cases
+See the [Use Cases README](use_cases/README.md) for info on our target use cases.
+
 # Requirements
-See the [requirements document](requirements/requirements.md) for the current list of requirements.
+See the [Requirements document](requirements/requirements.md) for the current list of requirements.
 
 # Design Overview
 See the [Navigation 2 Overview](design/Navigation_2_Overview.pdf) file for the current design / architecture
 
+# Differences from ROS Navigation
+See the [ROS_COMPARISON](design/ROS_COMPARISON.md) file for an overview of the differences between this design and ROS Navigation (move_base)
```

Comment:
I've been using "ROS 2" and "ROS 1" (or "ROS2" and "ROS1" without spaces) to be specific and "ROS" to be generic (could refer to ROS 1 or ROS 2 depending on context). So, I might say, "for an overview of the differences between this design and the ROS 1 Navigation stack design (move_base)."